exchanges/retry/src/retryExchange.ts

```diff
@@ -0,0 +1,118 @@
+import {
+  makeSubject,
+  share,
+  pipe,
+  merge,
+  filter,
+  tap,
+  fromValue,
+  delay,
+  mergeMap,
+  takeUntil,
+} from 'wonka';
+import { Exchange, Operation, CombinedError } from '@urql/core';
+
+interface RetryExchangeOptions {
+  initialDelayMs?: number;
+  maxDelayMs?: number;
+  randomDelay?: boolean;
+  maxNumberAttempts?: number;
+  // Optional function to determine whether an error should be retried
```

Comment:
```suggestion
  /** Conditionally determine whether an error should be retried */
```

^ This will make the prompt show up on your editor when you hover over it.